app/src/main/java/com/door43/translationstudio/tasks/ImportProjectsTask.java

```diff
@@ -0,0 +1,36 @@
+package com.door43.translationstudio.tasks;
+
+import com.door43.translationstudio.AppContext;
+import com.door43.util.tasks.ManagedTask;
+
+import java.io.File;
+
+/**
+ * Created by blm on 5/17/16.
+ */
+public class ImportProjectsTask extends ManagedTask {
+
+    public static final String TASK_ID = "imports_projects_task";
+
+    private File projectsFolder;
+    private boolean overwrite;
+    public boolean mSuccess;
```

Comment:
this boolean is never used. It would be proper to provide a public method that returns an array of target translations that were imported.